torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -94,15 +94,19 @@ def is_getitem_0_only_user(node):
     )
 
 
-def insert_nchw_to_nhwc(g, input_tensor):
-    return g.call_function(ttnn.permute, (input_tensor, (0, 2, 3, 1)))
+def insert_nchw_to_nhwc(g, input_tensor, input_shape):
+    return g.call_function(ttnn.permute, (input_tensor, (0, 2, 1) if len(input_shape) < 4 else (0, 2, 3, 1)))
 
 
 def insert_sharded_nhwc_to_nchw(g, output_tensor, output_shape):
-    batch_size, out_c, out_h, out_w = output_shape
+    is_1d = len(output_shape) < 4
```

Comment:
1d in the context of conv, but this method (at least from name) has nothing to do with convs. so its funny to read is_1d  = len < 4

Reply:
I'm not good at naming 😅 but I generalized these two helpers into `insert_ncx_to_nxc` and `insert_sharded_nxc_to_ncx` with some comments to explain the conversion